$OpenBSD: patch-src_config_h,v 1.2 2011/09/15 12:30:30 edd Exp $

Without this, saves go into user's home dirs, meaning that you don't get
a shared leader board. I thought upstream had already applied this (or
fixed it in some way), but I guess not.

--- src/config.h.orig	Sun Sep 11 19:51:10 2011
+++ src/config.h	Sun Sep 11 19:54:41 2011
@@ -52,11 +52,13 @@
  * OPTION: Create and use a hidden directory in the users home directory
  * for storing pref files and character dumps.
  */
+#if 0
 #ifdef SET_UID
 # ifndef PRIVATE_USER_PATH
 #  define PRIVATE_USER_PATH "~/.angband"
 # endif /* PRIVATE_USER_PATH */
 #endif /* SET_UID */
+#endif
 
 
 /*
